cmd/eskip/args.go

```diff
@@ -220,7 +220,7 @@ func processPatchArgs(pfilters, pfile, afilters, afile string) []*medium {
 	}
 
 	if afilters != "" {
-		media = append(media, &medium{typ: patchAppend, patchFilters: afilters})
+		media = append(media, &medium{typ: patchAppend, patchFilters: pfilters})
```

Comment:
ah.. sorry, my bad before, i didn't watch carefully enough. This basically reverts #354. Please keep it as 'afilters'.